vmodules/util/interfaces/BotConfig.ts

```diff
@@ -17,5 +29,69 @@ export default interface BotConfig {
   },
   api: {
     port: number
-  }
+  },
+  colors: {
+    default: ColorResolvable
```

Comment:
i feel like there should be more to the colors object here.

Reply:
where are the error/warn/okay colors?